### PR TITLE
Discord Permissions Calculator

### PR DESCRIPTION
**Pull request by Limerio**
I have add a website status it's Discord Permissions Calculator

### CODE REVIEW

File: websites/D/Discord Permissions Calculator/tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  "exclude": [
+    "node_modules"
+  ],
+  {
+    "extends": "../../../tsconfig.json",
+    "compilerOptions": {
+      "outDir": "./dist/"
+    }
+  }
+}
```
Comment (Bas950) [inline websites/D/Discord Permissions Calculator/tsconfig.json:16]: https://docs.premid.app/en/dev/presence#filling-in-the-tsconfigjson-file

File: websites/D/Discord Permissions Calculator/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.0",
+  "author": {
+    "name": "Limerio",
+    "id": "491841063057620995"
+  },
+  "service": "Discord Permissions Calculator",
+  "description": {
+    "en": "Calculator permissions of your bot and convert in link for your bot and add on your server.",
```
Comment (Bas950) [inline websites/D/Discord Permissions Calculator/dist/metadata.json:9]: Calculate*

**Comment by Limerio** [comment]
What proof dis you get ? A screen ?

**Comment by Bas950** [comment]
yea screenshot of what it shows in discord

File: websites/D/Discord Permissions Calculator/tsconfig.json [hunk 3]
```diff
@@ -0,0 +1,8 @@
+{
+   {
+    "extends": "../../../tsconfig.json",
+    "compilerOptions": {
+      "outDir": "./dist/"
+    }
+  }
+}
```
Comment (doomlerd) [inline websites/D/Discord Permissions Calculator/tsconfig.json:8]: still wrong

**Comment by frozaum** [comment]
Closing this in 2 days if the screenshots/changes are presented

**Pull request closed without merging**